tasks.py

```diff
@@ -429,18 +436,15 @@ def PeriodicTask(interval=0, cronName="default"):
 		:param interval: Call at most every interval minutes. 0 means call as often as possible.
 		:type interval: int
 	"""
-
 	def mkDecorator(fn):
+		global _periodicTasks
 		if fn.__name__.startswith("_"):
 			raise RuntimeError("Periodic called methods cannot start with an underscore! "
 							   f"Please rename {fn.__name__!r}")
-		global _periodicTasks, _periodicTaskID
 		if not cronName in _periodicTasks:
 			_periodicTasks[cronName] = {}
 		_periodicTasks[cronName][fn] = interval
-		fn.periodicTaskID = _periodicTaskID
-		fn.periodicTaskName = "%s_%s" % (fn.__module__, fn.__name__)
-		_periodicTaskID += 1
+		fn.periodicTaskName = ("%s_%s" % (fn.__module__, fn.__qualname__)).replace(".", "_").lower()
```

Comment:
I thought we've agreed to use format-strings :)
```suggestion
		fn.periodicTaskName = f"{fn.__module__}_{fn.__qualname__}".replace(".", "_").lower()
```

Reply:
Nope

Reply:
It would be faster, but okay - it will works this way as well...